internal/cli/atlas/livemigration/live_migration.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package livemigration
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/atlas/livemigration/link"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
```

Comment:
for this and the create builder, can you please add a test similar to the one you fixed for the Atlas one, we try to test subcommands and flags to make sure we have the right ones

Reply:
Will do!